### PR TITLE
[es] feature: demo structure - spanish translation

### PR DESCRIPTION
**Pull request by krol3**
Related the issue https://github.com/open-telemetry/opentelemetry.io/issues/4887

### CODE REVIEW

**Comment by krol3** [comment]
@ramrodo @theletterf @electrocucaracha, could you review please?

File: content/es/docs/demo/_index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  repo: https://github.com/open-telemetry/opentelemetry-demo
+weight: 180
+cSpell:ignore: OLJCESPC
+---
+
+Bienvenido a la documentación de la [Demo de OpenTelemetry](/ecosystem/demo/), que
+cubre cómo instalar y ejecutar la demostración, y algunos escenarios que puede usar para ver
+OpenTelemetry en acción.
+
+## Ejecución la Demo
```
Comment (ramrodo) [inline content/es/docs/demo/_index.md:14]:
```suggestion
## Ejecución de la Demo
```

File: content/es/docs/demo/_index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+title: Documentación de la demo de OpenTelemetry
+linkTitle: Demo
+cascade:
+  repo: https://github.com/open-telemetry/opentelemetry-demo
+weight: 180
+cSpell:ignore: OLJCESPC
+---
+
+Bienvenido a la documentación de la [Demo de OpenTelemetry](/ecosystem/demo/), que
+cubre cómo instalar y ejecutar la demostración, y algunos escenarios que puede usar para ver
```
Comment (ramrodo) [inline content/es/docs/demo/_index.md:11]:
```suggestion
cubre cómo instalar y ejecutar la demostración, y algunos escenarios que puedes usar para ver
```

File: content/es/docs/demo/_index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+| Python     | [Servicio de Recomendaciones](services/recommendation/)        |                                                                                                   | [Servicio de Recomendaciones](services/recommendation/)                                           |
+| Ruby       |                                                                | [Servicio de Correo Electrónico](services/email/)                                                 | [Servicio de Correo Electrónico](services/email/)                                                 |
+| Rust       |                                                                | [Servicio de Envío](services/shipping/)                                                           | [Servicio de Envío](services/shipping/)                                                           |
+
+
+## Documentación de los Servicios
+
+Puede encontrar información específica sobre cómo se implementa OpenTelemetry en cada servicio aquí:
```
Comment (ramrodo) [inline content/es/docs/demo/_index.md:41]:
```suggestion
Puedes encontrar información específica sobre cómo se implementa OpenTelemetry en cada servicio aquí:
```

File: content/es/docs/demo/_index.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+- [Servicio de Pagos](services/payment/)
+- [Servicio de Catálogo de Productos](services/product-catalog/)
+- [Servicio de Cotizaciones](services/quote/)
+- [Servicio de Recomendaciones](services/recommendation/)
+- [Servicio de Envío](services/shipping/)
+- [Servicio Proveedor de Imágenes](services/imageprovider/)
+
+## Escenarios
+
+¿Cómo se pueden resolver los problemas con OpenTelemetry? Estos escenarios le muestran
```
Comment (ramrodo) [inline content/es/docs/demo/_index.md:59]:
```suggestion
¿Cómo se pueden resolver los problemas con OpenTelemetry? Estos escenarios te muestran
```

File: content/es/docs/demo/_index.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+algunos problemas preconfigurados y cómo interpretar los datos de OpenTelemetry para
+resolverlos.
+
+Agregaremos más escenarios con el tiempo.
+
+- Generar un [Error de Catálogo de Producto](feature-flags) para solicitudes `GetProduct`
+con ID de producto: `OLJCESPC7Z` utilizando el servicio Feature Flag
+- Descubra una fuga de memoria y diagnostíquela mediante métricas y seguimientos.
```
Comment (ramrodo) [inline content/es/docs/demo/_index.md:67]:
```suggestion
- Descubre una fuga de memoria y diagnostíquela mediante métricas y seguimientos.
```

**Review by ramrodo — COMMENTED** [review]
Some suggestions made.

File: content/es/docs/demo/_index.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+title: Documentación de la demo de OpenTelemetry
+linkTitle: Demo
+cascade:
+  repo: https://github.com/open-telemetry/opentelemetry-demo
+weight: 180
+cSpell:ignore: OLJCESPC
+---
+
+Bienvenido a la documentación de la [Demo de OpenTelemetry](/ecosystem/demo/), que
+cubre cómo instalar y ejecutar la demostración, y algunos escenarios que puedes usar para ver
```
Comment (theletterf) [inline content/es/docs/demo/_index.md:12]:
```suggestion
describe cómo instalar y ejecutar la demostración, además de algunos escenarios que puedes usar para ver
```

It's OK to depart lightly from the original text to convey the same meaning.

File: content/es/docs/demo/_index.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+OpenTelemetry en acción.
+
+## Ejecución de la Demo
```
Comment (theletterf) [inline content/es/docs/demo/_index.md:15]:
```suggestion
## Ejecución de la demo
```

Title case is not a thing in Spanish, so all titles we found in EN that use title case should be converted to sentence case.

File: content/es/docs/demo/_index.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+¿Quieres implementar la demo y verla en acción? Comienza aquí.
+
+- [Docker](docker-deployment/)
+- [Kubernetes](kubernetes-deployment/)
+
+## Referencia de Funciones del Lenguaje de Programación
```
Comment (theletterf) [inline content/es/docs/demo/_index.md:22]:
```suggestion
## Referencia de funciones del lenguaje de programación
```

File: content/es/docs/demo/_index.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+¿Quieres entender cómo funciona la instrumentación de un lenguaje en particular? Comienza aquí.
+
+| Lenguaje   | Instrumentación Automática                          | Bibliotecas de Instrumentación                                                                    | Instrumentación Manual                                                                       |
```
Comment (theletterf) [inline content/es/docs/demo/_index.md:26]:
```suggestion
| Lenguaje   | Instrumentación automática                          | Bibliotecas de instrumentación                                                                    | Instrumentación manual                                                                       |
```

File: content/es/docs/demo/_index.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+| PHP        |                                                                | [Servicio de Cotizaciones](services/quote/)                                                       | [Servicio de Cotizaciones](services/quote/)                                                       |
+| Python     | [Servicio de Recomendaciones](services/recommendation/)        |                                                                                                   | [Servicio de Recomendaciones](services/recommendation/)                                           |
+| Ruby       |                                                                | [Servicio de Correo Electrónico](services/email/)                                                 | [Servicio de Correo Electrónico](services/email/)                                                 |
+| Rust       |                                                                | [Servicio de Envío](services/shipping/)                                                           | [Servicio de Envío](services/shipping/)                                                           |
+
+
+## Documentación de los Servicios
```
Comment (theletterf) [inline content/es/docs/demo/_index.md:40]:
```suggestion
## Documentación de los servicios
```

File: content/es/docs/demo/_index.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+## Escenarios
+
+¿Cómo se pueden resolver los problemas con OpenTelemetry? Estos escenarios te muestran
+algunos problemas preconfigurados y cómo interpretar los datos de OpenTelemetry para
+resolverlos.
+
+Agregaremos más escenarios con el tiempo.
+
+- Generar un [Error de Catálogo de Producto](feature-flags) para solicitudes `GetProduct`
+con ID de producto: `OLJCESPC7Z` utilizando el servicio Feature Flag
```
Comment (theletterf) [inline content/es/docs/demo/_index.md:67]:
```suggestion
con ID de producto: `OLJCESPC7Z` utilizando el servicio Feature Flag.
```

File: content/es/docs/demo/_index.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+- Descubre una fuga de memoria y diagnostíquela mediante métricas y seguimientos.
```
Comment (theletterf) [inline content/es/docs/demo/_index.md:68]:
```suggestion
- Descubre una fuga de memoria y diagnostícala mediante métricas y seguimientos.
```

**Review by theletterf — CHANGES_REQUESTED** [review]
Excellent work! Just some comments to better align.

**Comment by theletterf** [comment]
@krol3 PTAL, thanks!

File: content/es/docs/demo/_index.md [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+
+## Referencia
+
+Documentación de referencia del proyecto, como requisitos y matrices de características.
+
+- [Arquitectura](architecture/)
+- [Desarrollo](development/)
+- [Referencia de Feature Flags](feature-flags/)
+- [Matriz de Características de Métricas](telemetry-features/metric-coverage/)
+- [Requisitos](./requirements/)
```
Comment (electrocucaracha) [inline content/es/docs/demo/_index.md:79]:
```suggestion
- [Requisitos](requirements/)
```

File: content/es/docs/demo/_index.md [hunk 14]
```diff
@@ -0,0 +1,84 @@
...
+## Ejecución de la Demo
+
+¿Quieres implementar la demo y verla en acción? Comienza aquí.
+
+- [Docker](docker-deployment/)
+- [Kubernetes](kubernetes-deployment/)
+
+## Referencia de Funciones del Lenguaje de Programación
+
+¿Quieres entender cómo funciona la instrumentación de un lenguaje en particular? Comienza aquí.
```
[the remark this answers is not in the document]
Comment (theletterf) [reply content/es/docs/demo/_index.md:24]: https://www.rae.es/espanol-al-dia/tilde-en-que-cuales-quienes-como-cuan-cuantoaosas-cuando-donde-y-adonde

**Review by theletterf — APPROVED** [review]

**Pull request merged**